translations/units.tex/pl.po

```diff
@@ -120,6 +120,21 @@ msgid ""
 "\\textbf{Reinforcing} requires that your Town has a Citadel in addition to a Dwelling of that Unit's Level.\\par\n"
 "\n"
 msgstr ""
+"\\hypertarget{Jednostki}{Wraz}\\index{Jednostki} z Bohaterem i powiązaną z nim Talią, gracze posiadają także \\textbf{Talię Kart Jednostek}, która reprezentuje armię poruszającą się z ich Bohaterami.\n"
+"Każda Frakcja ma dostęp do \\textbf{7 różnych Jednostek} o unikalnych zdolnościach i atrybutach.\n"
+"Jednostki są niezbędne do wygrywania Walk i realizacji celów Scenariusza.\n"
+"Każdy Scenariusz posiada instrukcje wskazującą, które Karty Jednostek powinny być zawarte w twojej początkowej Talii Jednostek.\n"
+"Talia ta powinna być oddzielona od reszty Jednostek twojej Frakcji.\\par\n"
+"Każda Jednostka Frakcji jest dwustronna, posiada słabszą stronę \"Garstka\" oraz silniejszą stronę \"Grupa\".\n"
+"Jednostki \"Garstka\" mogą być ulepszone do \"Grupa\" poprzez \\textbf{Wzmocnienie}, które podczas otrzymywania obrażeń mogą zostać zredukowane do strony \"Garstka\".\n"
+"Jednostki powinny być zawsze trzymane po właściwej stronie podczas ich przemieszczania lub sprawdzania.\\par\n"
+"Talia Jednostek gracza może zawierać dowolną liczbę Jednostek, ale \\textbf{tylko do 5 Jednostek} może zostać wybrane do Walki podczas \\hyperlink{Przygotowania Walki}.\n"
+"Jeśli Jednostka zostanie pokonana w Walce, \\textbf{odrzuć ją} z Talii Jednostek.\\par\n"
+"Jednostki mogą być Zrekrutowane\\index{Rekrutacja} oraz Ulepszone\\index{Wzmacnianie} odwracając Żeton Populacji\\index{Żeton Populacji} i płacąc Koszt Rekrutowania \\includesvg[height=10px]{\\svgs/pay.svg} lub Wzmocnienia \\includesvg[height=10px]{\\svgs/reinforce.svg}.\n"
```

Comment:
może tak będzie czytelniej

```suggestion
"Jednostki mogą być Zrekrutowane\\index{Rekrutacja} oraz Ulepszone\\index{Wzmacnianie} poprzez odwrócenie Żetonu Populacji\\index{Żeton Populacji} i opłacenie Kosztu Rekrutowania \\includesvg[height=10px]{\\svgs/pay.svg} lub Wzmocnienia \\includesvg[height=10px]{\\svgs/reinforce.svg}.\n"
```

Reply:
Ma to sens, mi też się lepiej czyta
Dodane